sensing/gnss_poser/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 3.14)
 project(gnss_poser)
 
 find_package(autoware_cmake REQUIRED)
+find_package(autoware_sensing_msgs REQUIRED)
```

Comment:
Probably this line is not necessary.

Reply:
> Probably this line is not necessary.

Yes i checked, you are right. Many thanks for your feedback.

Reply:
Fixed! https://github.com/autowarefoundation/autoware.universe/pull/1230/commits/ea54e317fbad0209f3d24be5c7b3678e214cdbe4